app/components/Social/SocialButton.jsx

```diff
@@ -0,0 +1,37 @@
+/* eslint indent: [2, 2, {"SwitchCase": 1}] */
+import React from 'react';
+
+const isBrowser = !(typeof document === 'undefined' || typeof window === 'undefined');
+
+function click(onClick, url, event) {
+  onClick && onClick(event);
+
+  if (isBrowser) {
+    window.open(url, '_blank');
+  }
+}
+
+function getUrl(service, message, url) {
+  const msg = message ? [message, url].join(' ') : url;
+
+  switch (service) {
+    case 'vkontakte':
+      return `http://vk.com/share.php?url=${encodeURIComponent(url)}&description=${encodeURIComponent(message)}`;
+    case 'facebook':
+      return `https://www.facebook.com/sharer/sharer.php?u=${encodeURIComponent(url)}`;
```

Comment:
Got it, makes sense. I've written to you at https://web-standards.slack.com, can we talk?